javascript/index.js

```diff
@@ -40,26 +42,69 @@ function clearSplits() {
 
 function setStopBtn() {
   // ... your code goes here
+  /*set*/ 
+    /*set*/ 
+
+ /*set*/ 
 }
 
 function setSplitBtn() {
   // ... your code goes here
+  
 }
 
 function setStartBtn() {
   // ... your code goes here
+  /*set*/
 }
 
 function setResetBtn() {
   // ... your code goes here
 }
 
 // Start/Stop Button
-btnLeftElement.addEventListener('click', () => {
+
   // ... your code goes here
-});
+  btnLeftElement.addEventListener('click', () => {
+
+  if(btnLeftElement.textContent === 'START' ) {
+    btnLeftElement.textContent = 'STOP';
+    btnRightElement.textContent = 'SPLIT';
+    chronometer.start(callback);
+  
+  } else  {
+    btnLeftElement.textContent = 'START';
+    btnRightElement.textContent = 'RESET';
+    chronometer.stop();
+
+  }
+
+  }
+
+
+
+);
 
 // Reset/Split Button
-btnRightElement.addEventListener('click', () => {
+
   // ... your code goes here
-});
+  btnRightElement.addEventListener('click', () => {
+    
+  if (btnRightElement.textContent === 'RESET') {
+    btnRightElement.textContent = 'SPLIT';
+    btnLeftElement.textContent = 'STOP';
+    chronometer.reset();
+  }
+  else {
+    btnRightElement.textContent = 'RESET';
+    btnLeftElement.textContent = 'START';
+    chronometer.split();
+
+  }
+
+
+
```

Comment:
I don't have much more to say since you didn't do that much.
We'll try to explain a few things in class later. The lab was hard, so you'd better go back and try to understand.